src/components/Tooltip/components/TooltipOverlay/TooltipOverlay.tsx

```diff
@@ -16,10 +17,15 @@ export interface TooltipOverlayProps {
   preferredPosition?: PositionedOverlayProps['preferredPosition'];
   children?: React.ReactNode;
   activator: HTMLElement;
+  accessibilityLabel?: string;
   onClose(): void;
 }
 
-export class TooltipOverlay extends PureComponent<TooltipOverlayProps, never> {
+type CombinedProps = TooltipOverlayProps & {
+  i18n: ReturnType<typeof useI18n>;
+};
+
+class TooltipOverlayInner extends PureComponent<CombinedProps, never> {
```

Comment:
Possibly volunteering you for bonus work, but it looks pretty straightforward to rewrite this to be a functional component

Reply:
I don't mind 😄  I'll have to move this to an enhancement rather than bug fix so it's not accidentally shipped though.